org/mozilla/jss/tests/Constants.java

```diff
@@ -54,7 +54,26 @@ public static final int stringToCipher(String sCipher ) {
         }
         
     }
-    
+
+    /**
+     * The following have been deprecated
+     *
+     * SSL3_RSA_WITH_RC4_128_MD5
+     * SSL3_RSA_WITH_RC4_128_SHA
+     * SSL3_DHE_RSA_WITH_3DES_EDE_CBC_SHA
+     * SSL3_DHE_DSS_WITH_3DES_EDE_CBC_SHA
+     * SL3_RSA_WITH_3DES_EDE_CBC_SHA
+     * SSL3_DHE_RSA_WITH_DES_CBC_SHA
+     * SSL3_DHE_DSS_WITH_DES_CBC_SHA
+     * SSL3_RSA_WITH_DES_CBC_SHA
+     * SSL3_RSA_WITH_NULL_SHA
+     * SSL3_RSA_WITH_NULL_MD5
+     * SSL3_RSA_WITH_RC4_128_MD5
+     * SSL3_RSA_WITH_3DES_EDE_CBC_SHA
+     * SSL3_RSA_WITH_DES_CBC_SHA
+     * SSL3_RSA_WITH_NULL_MD5
+     */
+
```

Comment:
Many do but I didn't because it was getting messy and thought some are needed for reasons of compatibility so I opted to just list them in a conventional comment.side note:  I think I saw somewhere  that jss now plays well the system wide crypto policy and will likely prevent usage of insecure cipher suites. These two seem to be the worst. Are you suggesting me to add the @Deprecatedannotation to only those two or all of them?

Reply:
I know `nss` supports system-wide crypto policy, but I'm not sure about `jss`. I've been meaning to take a look at that; I think the first step is creating a basic client/server, deploying that to a system, and then automating testing of various cipher suite configurations.